styles/octopus.css

```
@@ -0,0 +1,48 @@
+.octopus {
+    padding-bottom: 84px;
+
+    background-image:
+    url("../images/bubble.png"),
+    url("../images/Group_bubls.png"),
+    linear-gradient(252.43deg, #D45B78 0.01%, #4B3862 101.01%);
+    background-repeat: 
+    no-repeat,
+    no-repeat,
+    no-repeat;
+    background-position: 
+    left 128px bottom 109px,
+    right 27px bottom 31px,
+    top 0 left 0;
+}
+
+.octopus__container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
```

```suggestion
// 84px is padding-bottom of .octopus
```
Please add comments in confusing places, for code readability